.circleci/config.yml

```diff
@@ -462,3 +528,12 @@ workflows:
       - py39-integration-goethereum-ws
       - py39-integration-ethtester-pyevm
       - py39-wheel-cli
+      - py310-integration-ethtester-pyevm
```

Comment:
looks like `py310-integration-ethtester-pyevm` and `py310-wheel-cli` are on here twice